core/domain/topic_domain.py

```diff
@@ -561,7 +563,7 @@ def __init__(
         self.practice_tab_is_displayed = practice_tab_is_displayed
         self.page_title_fragment_for_web = page_title_fragment_for_web
 
-    def to_dict(self):
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
Remove the typing and add ignore to the callsite

Reply:
done!